substrate/substrate-test-runtime/src/lib.rs

```diff
@@ -999,6 +999,12 @@ cfg_if! {
                 ) -> Option<Vec<subspace_core_primitives::RootBlock>> {
                     panic!("Not needed in tests")
                 }
+
+                fn extract_block_object_mapping(
+                    _block: Block
+                ) -> subspace_core_primitives::objects::BlockObjectMapping {
+                    Default::default()
```

Comment:
Either this should be `subspace_core_primitives::objects::BlockObjectMapping::default()` or below should be `Default::default()` for consistency